apps/wrf/example/namelist.tpl.wps

```diff
@@ -0,0 +1,39 @@
+&share
```

Comment:
Is this example file useful? It alone cannot be used to generate a WRF input file?

Reply:
the namelist.wps file is used by the preparation tools and is needed to create the WRF input